server/app/assets/javascripts/enumerator.ts

```diff
@@ -4,20 +4,21 @@
 import {addEventListenerToElements, assertNotNull} from './util'
 
 export function init() {
-  // Configure the button on the enumerator question form to add more enumerator field options
-  const enumeratorOptionButton = document.getElementById(
-    'enumerator-field-add-button',
+  addEventListenerToElements(
+    '.cf-question-enumerator input[data-entity-input]',
+    'input',
+    refreshAddButtonStatus,
   )
-  if (enumeratorOptionButton) {
-    enumeratorOptionButton.addEventListener('click', addNewEnumeratorField)
-  }
+
+  document
```

Comment:
suggestion (non-blocking): You could also make this a call to `addEventListenerToElements` to have it be extra cute and match its friends.

Reply:
Oh yeah I thought about that but it's using getById to select a single element instead of a selector to select potentially many so I was thinking it was different. But of course I can make an ID selector... I guess might as well have it match its friends; the . vs # will make it clear they are different. Thanks for the suggestion!